library/rocket.chat

```diff
@@ -1,10 +1,14 @@
-# this file is generated via https://github.com/RocketChat/Docker.Official.Image/blob/0460cf4a47b5abe2fdc3500ceb63c8d3ceccd4f9/generate-stackbrew-library.sh
+# this file is generated via https://github.com/RocketChat/Docker.Official.Image/blob/b73be519a9f4b22e58905885790c17dbcca19bc7/generate-stackbrew-library.sh
 
 Maintainers: Rocket.Chat Image Team <buildmaster@rocket.chat> (@RocketChat)
 GitRepo: https://github.com/RocketChat/Docker.Official.Image.git
 
-Tags: 6.3.5, 6.3, 6, latest
-GitCommit: 42a6ffd7ceb1064ab4baf38e5733d41faab0519d
+Tags: 6.4.1, 6.4, 6, latest
+GitCommit: b73be519a9f4b22e58905885790c17dbcca19bc7
+Directory: 6.4
+
+Tags: 6.3.10, 6.3
+GitCommit: b73be519a9f4b22e58905885790c17dbcca19bc7
 Directory: 6.3
 
 Tags: 6.2.12, 6.2
```

Comment:
Any particular reason to keep these older versions (`5.4` and `6.2`) listed here? They seem to have reached end of life (https://docs.rocket.chat/resources/rocket.chats-support-structure/enterprise-support-and-version-durability.

Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Apologies, I think I was waiting for an update to this PR.  Should I merge this PR to get the new version in or did you want to update it to drop the EOL version?

Reply:
my bad @yosifkit .. I've updated it now removing 5.4 .. thx

Reply:
No worries. I was making sure that you weren't waiting on me to merge in order to do it in a separate PR (either was fine by me).